lib/widgets/inbox.dart

```diff
@@ -246,7 +257,7 @@ abstract class _HeaderItem extends StatelessWidget {
               overflow: TextOverflow.ellipsis,
               title))),
           const SizedBox(width: 12),
-          // TODO(#384) for streams, show @-mention indicator when it applies
+          if (hasMention) const _AtMentionMarker(),
```

Comment:
nit in commit message:

    inbox: Show at-mention marker on topics

Should say "topics and streams", since the same commit does it for streams too.